qemu/tests/cfg/rng_bat.cfg

```diff
@@ -4,13 +4,17 @@
     virt_test_type = qemu
     read_rng_timeout = 360
     no no_virtio_rng
+    pre_cmd = while true; do dd if=/dev/random of=/dev/null bs=1 count=10; done
```

Comment:
Does the command line run on host backend, right?
I don't find the place quote it.